test/Microsoft.TemplateEngine.TestHelper/PackageManager.cs

```diff
@@ -21,14 +21,21 @@ public class PackageManager : IDisposable
     {
         private const string NuGetOrgFeed = "https://api.nuget.org/v3/index.json";
         private static readonly SemaphoreSlim Semaphore = new SemaphoreSlim(1, 1);
-        private readonly string _packageLocation = TestUtils.CreateTemporaryFolder("packages");
+        private string _packageLocation = TestUtils.CreateTemporaryFolder("packages");
         private readonly ConcurrentDictionary<string, string> _installedPackages = new ConcurrentDictionary<string, string>(StringComparer.OrdinalIgnoreCase);
         private static readonly Mutex PackMutex = new Mutex(false, "TemplateEngineTestPackMutex");
 
-        public async Task<string> GetNuGetPackage(string templatePackName, string? version = null, NuGetVersion? minimumVersion = null, ILogger? logger = null)
+        public string PackageLocation
+        {
+            get => _packageLocation;
+            set => _packageLocation = value;
```

Comment:
could you please reconsider if we need this setter? 
Since `downloadDirectory` is available in `GetNuGetPackage` now, it looks like it is not needed anymore.

Reply:
I forgot to remove the change. Thanks for catching it!